GitLink.py

```diff
@@ -123,7 +126,8 @@ def run(self, edit, **args):
         remote_path = self.getoutput("git rev-parse --show-prefix")
 
         # Find the current revision
-        if 'revision_type' in args and args['revision_type'] == 'commithash':
+        settings = sublime.load_settings("Preferences.sublime-settings")
+        if settings.get('gitlink_revision_type') == 'commithash':
```

Comment:
I'm not sure if the `args['revision_type']` is old syntax from Sublime 2 or not, but the value was not being passed into the `args` as far as I could tell rendering it ineffective. Seems like the way to do this with Sublime 3 is to read the settings values from `load_settings`.